components/tidb_query_normal_expr/src/builtin_string.rs

```diff
@@ -2250,6 +2275,90 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_make_set() {
+        let cases = vec![
```

Comment:
Could you add a case of 0 first argument? Rest LGTM.

Reply:
Sure. I have added test case.